### PR TITLE
Scale Out Repos

### PR DESCRIPTION
**Pull request by berndleinfelder**
created patch to support monitoring of scale out repo extents

### CODE REVIEW

**Comment by eren-kemer** [comment]
No errors this time.

**Comment by mycloudrevolution** [comment]
I have not forgot the Pull Request...
I think I can test ist soon. My Lab is now ready again

File: Veeam/PRTG-VeeamBRStats.ps1 [hunk 1]
```diff
@@ -205,7 +238,7 @@ Write-Host "<result>"
                "</result>" 
 $Count = $failsSessionsBk.Count
 Write-Host "<result>"
-               "<channel>Failes-Backups</channel>"
```
Comment (mycloudrevolution) [inline Veeam/PRTG-VeeamBRStats.ps1]: Renaming the XML Sections is a problem for existing User.
Please remove all these Changes.

File: Veeam/PRTG-VeeamBRStats.ps1 [hunk 2]
```diff
@@ -127,6 +146,20 @@ If ($reportMode -eq "Monthly") {
...
+
+if ($scaleouts) {
+    foreach ($scaleout in $scaleouts) {
+        $extents = Get-VBRRepositoryExtent -Repository $scaleout
+        foreach ($ex in $extents) {
+            $repoList = $repoList + $ex.repository
+        }
+    }
+}
+
```
Comment (mycloudrevolution) [inline Veeam/PRTG-VeeamBRStats.ps1]: Only the Extends are reported not the Repo itself. Is that by design?

File: Veeam/PRTG-VeeamBRStats.ps1 [hunk 3]
```diff
@@ -36,6 +35,26 @@ param(
...
+    }else{
+        &"$env:WINDIR\sysnative\windowspowershell\v1.0\powershell.exe" -NonInteractive -NoProfile -file "$($myInvocation.InvocationName)" $args
+    }
+exit $lastexitcode
+}
+
+
+
+
+
```
Comment (mycloudrevolution) [inline Veeam/PRTG-VeeamBRStats.ps1]: Section does not work as expected. 

If executed on 32Bit PowerShell Promt, Code is uncompleted.
`PS D:\> & '.\PRTG Veeam Advanced Sensor.ps1
>>
>>
>>
>>
>>`

**Comment by mycloudrevolution** [comment]
@berndleinfelder might you please check the review?

If not I will do my own Commit for the Scale Out Repo Part and close your Pull Request.

**Pull request closed without merging**